projects/robots/universal_robots/worlds/ure.wbt

```diff
@@ -6,7 +6,6 @@ WorldInfo {
   title "Universal Robot"
   basicTimeStep 8
```

Comment:
Overall is fine, there's just some minor discrepancies with the master version:

This branch:
![a](https://user-images.githubusercontent.com/44834743/134858471-20b4acf3-374b-48fa-9fa8-c534147ee5a1.png)
![b](https://user-images.githubusercontent.com/44834743/134858489-949ac357-1ffc-42d1-ae79-27a31d284bdb.png)

master:
![c](https://user-images.githubusercontent.com/44834743/134858531-afaf7418-437e-40ae-b16d-3b6574afdc61.png)

Technically these objects also need a 90° rotation:
![d](https://user-images.githubusercontent.com/44834743/134858571-f9c49fb3-262e-46e7-8400-63e812f0c364.png)
on master:
![e](https://user-images.githubusercontent.com/44834743/134858690-ae7308fe-2388-4212-9c01-aa5d5d762ba0.png)

Reply:
It should be fixed now